CONTRIBUTING.md

```diff
@@ -70,15 +70,22 @@ https://packaging.python.org/guides/distributing-packages-using-setuptools/#pack
   ```
 
 - Update version in setup.py, replace "unreleased" heading in changelog with the version number, and commit.
```

Comment:
I noticed you also rearrange into, Fixed, Changed and Added, should that be done at this step, or should we encourage all of us to do this when adding to the change log?

Reply:
Good point. I've been going off the advice in https://keepachangelog.com/en/1.0.0/#how. I'll add a note here.

There's also #355 for general improvements to the changelog process.